### DISCUSSION TITLE
Powiadomienia System Warnings na maila

### DISCUSSION THREAD

**Discussion by odiiin** [YetiForceCompany/YetiForceCRM#13995] [How to] [answered]
Witam,
Jak wybrać odbiorców do których powiadomienia System Warnings będą wysyłane? Teraz wydaje mi się, że wysyłane są do wszystkich administratorów a chciałbym żeby tylko do jednej osoby. Pozdrawiam

**Comment by rskrzypczak** [comment, marked as answer]
Wystarczy ustawić tylko jednego użytkownika administratorem :)
Jak słusznie zauważyłeś, powiadomienia tego typu wysyłane są do wszystkich adminów i nie jest to konfigurowalne.
